package-lock.json

```diff
@@ -6017,8 +6017,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
Can you do `nvm use` before `npm i` and see if we get these?

Reply:
Fixed.